analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseUriInsteadOfString.CSharp10.cs

```diff
@@ -16,7 +16,7 @@ public record struct S3994
     public S3994(string uri, bool blah) { } // Noncompliant {{Either change this parameter type to 'System.Uri' or provide an overload which takes a 'System.Uri' parameter.}}
 }
 
-public record struct WithParams(string uri) // FN
+public record struct WithParams(string uri) // Noncompliant {{Change this property type to 'System.Uri'.}}
```

Comment:
Can we check for the actual RuleId that is triggered here? Would be useful because there is more than one reported by this analyzer.

Reply:
No, we don't have a format for that. We can only assert the message. That's why added it this time.